tests/monitoring/component_monitoring.go

```diff
@@ -21,6 +21,7 @@ package monitoring
 
 import (
 	"context"
+	"kubevirt.io/kubevirt/tests/libvmi"
```

Comment:
please, move the import in the `kubevirt.io` section below

Reply:
Thanks :)